doc/notebooks/Molecular_Grid.ipynb

```diff
@@ -191,37 +198,81 @@
         "\n",
         "# Integrate using the molecular grid\n",
         "print(f\"Integration of the electron density: {mol_grid.integrate(electron_density)}.\")"
-      ],
-      "metadata": {
-        "collapsed": false,
-        "ExecuteTime": {
-          "end_time": "2023-05-18T16:48:05.389647782Z",
-          "start_time": "2023-05-18T16:48:04.376467773Z"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "### Local properties\n",
+        "\n",
+        "##### Example: Atomic charges from molecular electron density\n",
+        "\n",
+        "This example shows how to get extract grids from a molecular grid and integrate the electron density to get atomic charges. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 34,
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Molecular grid has 16448 points.\n",
+            "Oxygen grid has 4980 points.\n",
+            "Integration of the Oxygen atom electron density on the Oxygen grid: 6.213950819473563.\n",
+            "Atomic charge of the Oxygen atom: -0.21395081947356331.\n"
+          ]
         }
-      }
+      ],
+      "source": [
+        "# Extract the atomic grid of the Oxygen atom\n",
+        "o_grid, o_idcs = mol_grid.get_atomic_grid(1)\n",
+        "\n",
+        "print(f\"Molecular grid has {len(mol_grid.points)} points.\")\n",
+        "print(f\"Oxygen grid has {len(o_grid.points)} points.\")\n",
+        "\n",
+        "# Extract electron density on the Oxygen grid points\n",
+        "o_electron_density = electron_density[o_idcs] * mol_grid.aim_weights[o_idcs]\n",
```

Comment:
@marco-2023, I was thinking that it woul be more straightforward if `mol_grid.aim_weights[o_idcs]` were already included in the `o_grid`. That is, `o_grid` is not just a simple atomic grid, but it is aware of `aim_weights`. Your thoughts? We can talk about this.